lit_gpt/pretrain.py

```diff
@@ -1,13 +1,7 @@
 # Copyright Lightning AI. Licensed under the Apache License 2.0, see LICENSE file.
 
-"""
-This script is adapted from TinyLlama:
-https://github.com/jzhang38/TinyLlama/blob/main/pretrain/tinyllama.py
-"""
```

Comment:
While the script has changed a lot, there are still some artifacts left over, for example the weight initialization. So maybe keep this?

Reply:
What if we reference what hasn't changed? I can link tinyllama in the weight initialization function